clients/fides-js/src/lib/consent-constants.ts

```diff
@@ -123,6 +123,12 @@ export const FIDES_OVERRIDE_OPTIONS_VALIDATOR_MAP: FidesOverrideValidatorMap[] =
       overrideKey: "fides_modal_default_view",
       validationRegex: /^\/tcf\/(purposes|features|vendors)$/,
     },
+    {
+      overrideName: "fidesDefaultShowModal",
+      overrideType: "boolean",
+      overrideKey: "fides_default_show_modal",
+      validationRegex: /^(true|false)$/,
+    },
```

Comment:
Kind of nit-picky, but Tom has taught me the benefit of avoiding boolean as a setting value. Can we change this to be something like `fidesModalDisplay` with values like `"immediate"|"banner_click"` where the default is banner click? If we'd have done this with fides_embed originally, we could just add an option to that instead of creating a whole new override. It becomes much more extensible.